src/biocommons/seqrepo/fastadir/fastadir.py

```diff
@@ -50,13 +51,18 @@ class FastaDir(BaseReader, BaseWriter):
 
     """
 
-    def __init__(self, root_dir, writeable=False, check_same_thread=True, fd_cache_size=0):
+    def __init__(
+        self,
+        root_dir: str,
+        writeable: bool = False,
+        check_same_thread: bool = True,
+        fd_cache_size: Optional[int] = 0,
```

Comment:
Should this just be an int?
```suggestion
        fd_cache_size: int = 0,
```

Reply:
cache size of `None` means infinite, see https://github.com/biocommons/biocommons.seqrepo/commit/4d1a349e14374761ebc835339f66c4e9d7e97fc3

Reply:
Ah thank you!